### ISSUE TITLE
Seperate process for logs zip in runner load

### ISSUE THREAD

**Issue by itaiag** [Top-Q/jsystem#128]
Reported by: itzhak.hovav
Opened 17 months ago
Add the option for "parallel logs zip" on runner start: 
Problem description : sometimes, while the runner starts with many reports in the "current" directory from last run, it takes allot of time (sometimes its more than 30 minutes) for the runner to start because it zips the old log. 
Solution offered : Add the option to perform this task in a separate process with a lower priority. the runner will rename the "current" folder's name, open the zip process to run in parallel on the renamed folder and will immediately continue as it have no logs...
